virion.yml

```diff
@@ -1,6 +1,6 @@
 name: pmforms
 version: 1.0.0
 antigen: dktapps\pmforms
-api: 3.4.0
+api: [4.0.0, 4.0.0-BETA11, 4.0.0-BETA12]
```

Comment:
These are redundant. Only `4.0.0` is needed.

Reply:
I want to make sure it's still usable in beta version

Reply:
They are still redundant...